src/components/UploadShow/index.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { observer } from 'mobx-react';
+import classnames from 'classnames';
+import { translate } from 'react-i18next';
+
+import { Icon } from 'components/Base';
+
+import styles from './index.scss';
+
+@translate()
+@observer
+export default class UploadShow extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    errorMessage: PropTypes.string,
+    fileName: PropTypes.string,
+    isLoading: PropTypes.bool,
+    uploadStatus: PropTypes.string
+  };
+
+  render() {
+    const {
+      className,
+      isLoading,
+      errorMessage,
+      uploadStatus,
+      fileName,
+      t
+    } = this.props;
+
+    if (isLoading) {
+      return (
+        <div className={classnames(className, styles.uploading)}>
+          <Icon name="loading" size={48} type="dark" className={styles.icon} />
+          <p className={styles.note}>{t('file_format_loading')}</p>
+        </div>
+      );
+    }
+
+    return (
+      <div
+        className={classnames(className, styles.uploadShow, {
```

Comment:
Since `className` is an override style class, place it at last is more accurate.
```
classnames(styles.uploadShow, {...}, className)
```

Reply:
ok